frame/support/src/lib.rs

```diff
@@ -1577,15 +1581,11 @@ pub mod pallet_prelude {
 /// 	}
 ///
 /// 	// Define the pallet struct placeholder, various pallet function are implemented on it.
-/// 	// The macro checks struct generics: is expected `T` or `T, I = DefaultInstance`
 /// 	#[pallet::pallet]
 /// 	#[pallet::generate_store(pub(super) trait Store)]
 /// 	pub struct Pallet<T>(PhantomData<T>);
 ///
 /// 	// Implement on the pallet hooks on pallet.
```

Comment:
This phrase isn't clear to me; should it be "Implement the pallet hooks"? Perhaps adding a few examples of what these hooks are would be good too?

Reply:
yeah in my head it meant "implement the pallet hooks on `Pallet` struct. Though I just rewrite to "Implement the pallet hooks."
And added a bit more of context about what `Hooks` trait is.